doc/requirements.txt

```diff
@@ -32,4 +32,4 @@ lxml_html_clean
 --extra-index-url https://test.pypi.org/simple/
 pennylane-lightning-kokkos==0.42.0-dev11
 pennylane-lightning==0.42.0-dev11
-pennylane==0.42.0-dev19
+pennylane==0.42.0-dev22
```

Comment:
this version should be updated before the merge

Reply:
Done in [1d29b6f](https://github.com/PennyLaneAI/catalyst/pull/1619/commits/1d29b6fb0fc1fe1fe23b2e364fc920141d30cb26)